templates/taxi/driver_list.html

```diff
@@ -8,6 +8,9 @@ <h1>Driver list</h1>
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+        {% if driver.username == user.username %}
```

Comment:
The conditional check to identify the current user is correctly implemented. Ensure that the `user` context variable is passed to the template to make this comparison possible.